### PR TITLE
[MODULAR] [READY] Blueshield additions, fixes, and drip!

### PR DESCRIPTION
**Pull request by CandleJaxx**
## About The Pull Request

adds a number of helpful additions and changes to blueshields- including two new clothing options (a skirt, and an armor vest) to give them more variety in clothing and/or aesthetic

this also adds one more magazine of lynx ammo to the gunset, putting it in-line with other gunsets that are given four mags, wherein the lynx was previously only given three (presumably as an oversight)

aaand it also adds a new job title, 'Asset Retention Agent,' because it sounds cool and that's what i've been calling my blueshields anyway, because it's nice :>

## Images

image 
image
image
image
image

## How This Contributes To The Skyrat Roleplay Experience

giving blueshields more clothing options (and rounding out their gunset) will help their playability in the long run, as well as open up avenues for new / interesting character types

## Changelog
:cl:
expansion: Blueshields aboard NT frontier stations have been supplied replacement armor vests.
expansion: Blueshields have also been given skirts, for those who prefer that type of clothing.
qol: NanoTrasen recently busted an internal fraud ring wherein LYNX Gunset Magazines were being short-ended to be sold to outside companies. This has been mended.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/blueshield/code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -8,3 +8,13 @@
 	icon_state = "blueshield"
 	body_parts_covered = CHEST|ARMS
 	armor = list("melee" = 35, "bullet" = 25, "laser" = 25,"energy" = 25, "bomb" = 30, "bio" = 0, "rad" = 0, "fire" = 75, "acid" = 75)
+
+/obj/item/clothing/suit/armor/vest/blueshieldarmor
+	name = "Blueshield's Armor"
+	desc = "A tight-fitting kevlar-lined vest with a golden badge on the chest and \"NT\" emblazoned on the back. It weighs surprisingly little, despite how heavy it looks."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
+	//alternate_worn_icon_digi = 'modular_skyrat/icons/mob/suit_digi.dmi'
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/blueshield/code/modules/clothing/suits/armor.dm]: What was this comment for?

File: modular_skyrat/modules/blueshield/code/modules/clothing/under/jobs/security.dm [hunk 2]
```diff
@@ -22,3 +22,15 @@
...
 	random_sensor = FALSE
+
+/obj/item/clothing/under/rank/security/blueshieldskirt
+	desc = "A tactical skirt, seemingly outfitted in NanoTrasen's standard corporate-chic."
+	name = "Blueshield's skirt"
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
+	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/uniform_digi.dmi'
+	icon_state = "blueshieldskirt"
+	armor = list("melee" = 10, "bullet" = 5, "laser" = 5,"energy" = 10, "bomb" =10, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/blueshield/code/modules/clothing/under/jobs/security.dm]: If you can, do the same for the other ones in this file, this triggers my OCD.
```suggestion
	armor = list("melee" = 10, "bullet" = 5, "laser" = 5,"energy" = 10, "bomb" = 10, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
```

File: modular_skyrat/modules/blueshield/code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 3]
```diff
@@ -22,3 +22,5 @@
 	new /obj/item/clothing/suit/armor/vest/blueshield(src)
 	new /obj/item/clothing/shoes/laceup(src)
 	new /obj/item/clothing/under/rank/security/blueshield(src)
+	new /obj/item/clothing/suit/armor/vest/blueshieldarmor(src)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/blueshield/code/game/objects/structures/crates_lockers/closets/secure/security.dm:25]: I would probably move this one to the Command vendor, at the very least. Actually, most of the Blueshield's stuff should be moved to the Command vendor.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
It's alright, just a few things to tweak and we'll be good.

File: modular_skyrat/modules/blueshield/code/modules/clothing/suits/armor.dm (hunk 1 above)
Comment (CandleJaxx) [reply modular_skyrat/modules/blueshield/code/modules/clothing/suits/armor.dm]: comment was copied directly from the above obj, so i left it there just incase

Comment (GoldenAlpharex) [reply modular_skyrat/modules/blueshield/code/modules/clothing/suits/armor.dm]: > comment was copied directly from the above obj, so i left it there just incase

Then remove it ;)

**Comment by capsaicinz** [comment]
this is totally a great opportunity to mention the blueshield combat medkit..

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to go.

**Pull request merged**